### ISSUE TITLE
Feature Request: Add/Remove quantity for Accessories and Consumables

### ISSUE THREAD

**Issue by colinator19** [snipe/snipe-it#5173]
Right now, you can either adjust the total quantity and add/subtract the amount you are adding or removing OR add a "new" accessoire or consumable per every new order you place with the supplier but it seems not logical to do it this way, at least for me.

Would it be an idea if you can insert a option to add/remove a certain quantity so you can just add any quantity to the existing item when you buy again from the supplier or remove it when you are throwing some away?

**Comment by BHollowell** [comment]
I would also like to see this added. It would be nice to be able to quickly deduct, for example, 50 Ethernet cables from the consumables stock. Or add them quickly when new stock is ordered.

**Comment by absolutz23** [comment]
+1 to this, from what I see the consumable/accessories is like a stock inventory as the quantity will keep on changing and we still need to keep track the movement of the item. not sure how to do this. 👍

**Comment by Mynty100** [comment]
Clearly these features are well sort after. love to see them in upcoming releases.

**Comment by smatz1811** [comment]
We also have a similar need for this. We re-order prior to stock being completed so a simple adjust quantity button would allow us to use the existing record in cases where the exact same item has been ordered, and eliminate the possibility of slight variations in the product name. Being able to run a report on the number of re-orders for a given item would also be useful at the end of the fiscal year.

**Comment by Mynty100** [comment]
Yes, this is still very much required as we can't use this part of SNIPEIT as its current way of functioning does not work for our purposes so we are still using excel document to track printer ink etc.

**Comment by Mynty100** [comment]
Yes! still very much looking to have this added. we cant track printer toner properly with the way this is currently working. Cheers.

**Comment by colinerd** [comment]
Still relevant - looking to have this added in a future release.

**Comment by colinerd** [comment]
Still a very much wanted request :)

**Comment by tymik** [comment]
+1 for this and i see that #5176 is duplicate of this one!

**Comment by Mynty100** [comment]
Yes we cant use SNIPEIT to track printer toner purchases. Please consider this.
Thanking you,

**Comment by compuvin** [comment]
Going to +1 this too. I'm brand new to Snipe-IT but may be able to help out with pull requests in the future. :)

**Comment by Leofravega** [comment]
Same here, I would love to see this implemented. Please keep it open

**Comment by Mynty100** [comment]
Just wondering if anything is being done at all with this feature? at this stage, my I.T team finds it strange how this works as we have about 20 different toners. Please advise.
Thanking you,
Mynty.

**Comment by riptex** [comment]
This is a required feature. Right? It's gone stale so many times. Not sure what needs to happen?

**Comment by snipe** [comment]
@riptex A pull request. 

This is one of thousands of feature requests. Thanks for lending your voice here, but complaining only gets you so far.

**Comment by riptex** [comment]
Ah. Different perspective here; I'm paying for SnipeIT (Hosted) .. Wasn't treating it as OSS. Also, haven't even started complaining yet. Thanks for the quick response. You do NOT want me submitting pull requests. :)

**Comment by Mynty100** [comment]
I thank snipe for all the hard work on this project amd there team. Its ultimatly a free product that id be lost with out. 
So thank you for all the support regarding this web baeed application.

**Comment by AlxisHenry** [comment]
https://github.com/AlxisHenry/AUTOMATION_SNIPE-IT 

look this

**Comment by ChristianMichiels** [comment]
This is certainly a feature I would love to see. As we have inventory that we top up as the consumable becomes depleted. It seems odd to not have a way to just intake additional consumables to an already created item.

**Comment by allanPHD** [comment]
Same here, loving Snipe IT but this feature would be a big plus in day to day tracking!

**Comment by RussEfarmer** [comment]
This would help us out a lot

**Comment by samernasserbu** [comment]
still waiting for an answer! 
for example, we added a consumable (Toner HP 78A) from supplier X, quantity 20. when they are out of stock or nearly finished and need to reorder (this time from supplier Y), I need to add another instance of the 78A consumable, not add a new consumable!
Help, please. Thanks.

**Comment by wikilike7** [comment]
+1, please add a quantity for consumaables

**Comment by jonathanroots** [comment]
I strongly suggest to add quantity field when check out Consumables.

**Comment by janbiasi** [comment]
Any updates from contributors?

**Comment by ffish77** [comment]
+1

**Comment by mezka** [comment]
Need this feature

**Comment by afranchi74** [comment]
+1 for this feature.

**Comment by ottool** [comment]
+1! please implement this. otherwise snipe it is useless for our purposes

**Comment by afranchi74** [comment]
Any news abuot this request?

**Comment by etaxi341** [comment]
Any news? This is the most essential feature!

**Comment by krinkko** [comment]
Can we please seriously consider having this? We cannot really switch to SnipeIT without it. 

Right now I have like 14 entries for some Mouse + Keyboard Combo but it would be so neat just to work with a static Modell Number under which everything is resolved.

**Comment by afranchi74** [comment]
Still relevant - looking to have this added in a future release.

**Comment by EColichet** [comment]
Still relevant - i'm hoping there's adding this

**Comment by Geoduck111** [comment]
+1 

This would be really useful for my team. We are often taking consumables (e.g. batteries) out with us into the field, unsure if we will need to use all of them. Would be great to be able to add them back into stock if we don't use them on site.

**Comment by jonathanroots** [comment]
snipe did not update the app.

**Comment by boufet** [comment]
Is it planed on any further update? It really be helpful

**Comment by fantahund** [comment]
We could really use this feature as well. Having to navigate into the settings every single time just to adjust that number is incredibly tedious. We’re using this tool in a large corporate environment, so our team would be extremely grateful if this functionality could be added.

**Comment by sanitygaming** [comment]
+1. Crazy it has been so long since the suggestion.

**Comment by danielmilner** [comment]
The lack of this feature alone is what's preventing our company from moving forward with implementing Snipe-IT. I hope that it's available soon.

**Comment by TheXpertXD** [comment]
+100.. I feel like this has been one of the most requested features in this project, and has been around for years yet it seems like the team continues to downplay it as a minor issue. I understand there is a development process but you would think after 8 years and change there would have been some attempt from the SNIPE team by now. Numerous PRs denied as well, just disappointing to see

**Comment by snipe** [comment]
It *is* highly requested, but it's also really complicated. Numerous PRs were turned down because they didn't consider the backwards *and* forwards compatibility that we have to consider. 

>  there would have been some attempt from the SNIPE team by now.

There have been several. Flaws were found in each of them, and we've continued to work forward on the *right* path.

> just disappointing to see

And stuff like this is disappointing to read. We bust our asses every single day to provide a great free product, add new features, fix bugs, and move things forward, but none of that ever seems to matter.

**Comment by Mynty100** [comment]
Just want to start out by saying thank you Snipe. 
Thank you for the SnipeIT application it is amazing and great free tool for asset management.
This request upon further thought and reflection takes SnipeIT from being an asset management application and dangerously turns it into a **stock take** app. 
Which from what I can gather, is not what you and the SnipeIT team have ever set out to make.

It still would be a nice to have, to make the most use of consumables imo.

That said, if it is not something SnipeIT is going to support, then what is the current consumable's function and usability in you and your teams' eyes to be used for and how best to maintain it?
Is it worth removing? 
Should SnipeIT just focus 100% on asset side of things?

Honest questions and not meant to do anything more than help others here think levelheaded and start a clear and considerate conversation about the matter. All meant to be said with highest respect for you and your team.

At the end of the day. I love this product, I have used it since v3 or v4 days, around 2017 so almost 10yrs and nothing comes close.
So, thank you again for all the time and energy and passion that clearly gone into SnipeIT app.
